Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -26,6 +26,18 @@ public final class ReflectionService: CallHandlerProvider, Sendable {
     self.reflectionService.serviceName
   }
 
+  ///  Receives as parameter an array of strings representing the file paths to the binary files containing
+  ///  the serialised file descriptor protos describing the services supported by the Server and
+  ///  discovarable through Server Reflection.
+  public init(serializedFileDescriptorProtoFilePaths: [String]) throws {
```

Comment:
We should use a shorter label internally:

```suggestion
  public init(serializedFileDescriptorProtoFilePaths filePaths: [String]) throws {
```